Gemfile

```diff
@@ -92,6 +92,13 @@ group :development, :deployment do
   gem "capistrano-rails",   require: false
   gem "capistrano-rvm",     require: false
   gem "capistrano-bundler", require: false
+
+  # net-ssh requires the gems below to support ed25519 keys
+  # for deploying via capistrano
+  # more info at https://github.com/net-ssh/net-ssh/issues/478
+  gem "bcrypt_pbkdf", ">= 1.0", "< 2.0"
```

Comment:
Can we get away with `require: false` in these? And it's still not possible to remove the upper constraint? Looks like that issue was over a year ago.

Reply:
I’ll try specifying `require: false` here to see if `net-ssh` will pick them up correctly. As for the upper version constraint, we can’t get rid of it because even `net-ssh` 5 (and our bundle is still at 4.2) specifies the same `< 2.0` limit: https://github.com/net-ssh/net-ssh/issues/565

Reply:
@jhanggi `require: false` worked, so I added it in 4e71fa5 (along with mentioning that the note is for 4.2 of `net-ssh`)